skycultures/indian/constellation_names.eng.fab

```diff
@@ -76,6 +76,8 @@ PBhā	"Pūrva bhādrapada"	_("The first of the blessed feet")
 UBhā	"Uttara bhādrapada"	_("The second of the blessed feet")
 # Ultimately from रेव (प्लुतौ व्रजे).
 Rev	"Revatī"	_("Leaper")	
+# Secondary references: https://manasataramgini.wordpress.com/2004/12/16/comet-machholz-2004/
+Yam	"Yamunā"	_("Eridanus")
```

Comment:
It may be better to call it "YamunA River" in the international translation if the name "Eridanus" is not used by Indians.

Reply:
Done.